src/python/grpcio/grpc/_cython/_cygrpc/aio/server.pyx.pxi

```diff
@@ -822,7 +822,8 @@ cdef class AioServer:
         init_grpc_aio()
         # NOTE(lidiz) Core objects won't be deallocated automatically.
         # If AioServer.shutdown is not called, those objects will leak.
-        self._server = Server(options)
+        # TODO(rbellevi): Support xDS in aio server.
+        self._server = Server(options, False)
```

Comment:
Is there any blocker for supporting xDS config fetching in aio servers?

Reply:
Not in principle. In fact, I think it should be a very simple change. I just want to tackle that in a separate PR.